### PR TITLE
[Compose] Upgrade to Alpha 11

### PR DESCRIPTION
**Pull request by jossiwolf**
Upgrades Compose to Alpha 11!
I migrated some deprecated APIs, mostly related to effects. Also wrapped the player controls and toolbar in the `AnimatedVisibility` composable so toggling focus mode feels a little less jumpy.
Did some manual testing and everything looks good in the sample app 🎉 

Let me know what you think about the content descriptions for icons - I'll be happy to add some, but you tell me if it's needed!

### CODE REVIEW

File: sample-compose/src/main/java/com/airbnb/lottie/sample/compose/player/PlayerPage.kt [hunk 1]
```diff
@@ -105,12 +108,8 @@ fun PlayerPage(
         }
     }
 
-    onCommit(outlineMasksAndMattes.value) {
-        animationState.outlineMasksAndMattes = outlineMasksAndMattes.value
-    }
-    onCommit(applyOpacityToLayers.value) {
-        animationState.applyOpacityToLayers = applyOpacityToLayers.value
-    }
+    animationState.outlineMasksAndMattes = outlineMasksAndMattes.value
```
Comment (jossiwolf) [inline sample-compose/src/main/java/com/airbnb/lottie/sample/compose/player/PlayerPage.kt:111]: I am not exactly sure about this one - I don't think the `onCommit` was needed. Let me know what you think! Maybe this should actually be a `SideEffect`?

File: sample-compose/src/main/java/com/airbnb/lottie/sample/compose/lottiefiles/LottieFilesPage.kt [hunk 2]
```diff
@@ -74,8 +75,11 @@ fun LottieFilesTabBarTab(
 ) {
     val textWidth = remember { mutableStateOf(0) }
     val pxRatio = with(AmbientDensity.current) { 1.dp.toPx() }
-    val tabWidth by animateAsState(if (isSelected) (textWidth.value / pxRatio).dp else 0.dp, spring(), null)
-    val tabAlpha by animateAsState(if (isSelected) 1f else 0f)
+    val tabWidth by animateDpAsState(
+        if (isSelected) (textWidth.value / pxRatio).dp else 0.dp,
+        animationSpec = spring()
```
Comment (gpeal) [inline sample-compose/src/main/java/com/airbnb/lottie/sample/compose/lottiefiles/LottieFilesPage.kt:80]: What's wrong with the [default animation](https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:compose/animation/animation-core/src/commonMain/kotlin/androidx/compose/animation/core/AnimateAsState.kt;l=114?q=animateDpAsState)?

**Review by gpeal — COMMENTED** [review]
Looks good! Just one comment

File: sample-compose/src/main/java/com/airbnb/lottie/sample/compose/lottiefiles/LottieFilesPage.kt (hunk 2 above)
Comment (jossiwolf) [reply sample-compose/src/main/java/com/airbnb/lottie/sample/compose/lottiefiles/LottieFilesPage.kt:80]: Default animation is cool for me, but the `tabWidth` animation was using `spring()` before!

Comment (gpeal) [reply sample-compose/src/main/java/com/airbnb/lottie/sample/compose/lottiefiles/LottieFilesPage.kt:80]: Let's just stick with the default for simplicity!

Comment (jossiwolf) [reply sample-compose/src/main/java/com/airbnb/lottie/sample/compose/lottiefiles/LottieFilesPage.kt]: Sure, sounds good!

**Review by gpeal — APPROVED** [review]

**Pull request merged**